public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilder.tsx

```diff
@@ -125,12 +128,13 @@ export const LokiQueryBuilder = React.memo<Props>(
             labelsFilters={query.labels}
             onChange={onChangeLabels}
             labelFilterRequired={labelFilterRequired}
+            // multiValueSeparator="|"
```

Comment:
Maybe needs to be removed.

Reply:
👀  Fixed in https://github.com/grafana/grafana/pull/81140/commits/75ecb0948a5ba5a25e93a5d7ac320dcb525332d4